group_vars/all/git_repo_commit.yml

```diff
@@ -0,0 +1,11 @@
+---
+git_repo_commit: false
+
+github_organization: redhat-cop
+github_inventory_repo: rhis-inventory
+github_inventory_repo_path: "{{ github_organization }}/{{ github_inventory_repo }}"
+github_code_repo: rhis-code
+github_code_repo_path: "{{ github_organization }}/{{ github_code_repo }}"
+
+git_username: techuser-github-showroom
+git_email: bkaraore+showroom@redhat.com
```

Comment:
```suggestion
git_email: git@example.de
```

Reply:
updated email address, user should be there